src/NodeVisitor/NodeVisitorInterface.php

```diff
@@ -13,6 +13,9 @@
 
 use Twig\Environment;
 
+// Ensure that the aliased name is loaded to keep BC for classes implementing the typehint with the old aliased name.
+class_exists('Twig\Environment');
```

Comment:
Is it enough to add this to the `NodeVisitorInterface`, or should it also be added to implementations such as `AbstractNodeVisitor`? I'm asking because I found this PR because we were getting:

> Compile Error: Declaration of Translation\Bundle\Twig\Visitor\NormalizingNodeVisitor::doEnterNode(Twig_Node $node, Twig_Environment $env) must be compatible with Twig\NodeVisitor\AbstractNodeVisitor::doEnterNode(Twig\Node\Node $node, Twig\Environment $env)

Reply:
Also see https://github.com/php-translation/symfony-bundle/issues/288, we have the same issue there

Reply:
@Kocal that is indeed the bundle we are using, too :smile: I probably should have included the upstream issue / repository

Reply:
@nicwortel yes, it will be enough, as AbstractNodeVisitor implementing the interface will force the interface to be loaded.